src/lib/output/JunitOutputFormatter.ts

```diff
@@ -0,0 +1,84 @@
+import {OutputFormatter, Results} from "./Results";
+import {FormattedOutput, RuleResult, RuleViolation} from "../../types";
+import {isPathlessViolation} from "../util/Utils";
+import {safeHtmlEscape} from "./XmlOutputFormatter";
```

Comment:
It feels a little weird that this method is being imported from `XmlOutputFormatter`. I feel like it should be defined in some third place and they both import it from there? Maybe one of our existing `utils` classes, or perhaps a new one?

Reply:
Yeah I agree. I don't like poluting utils classes though for things that are only used twice. Once something is used three times, then it might make sense. Now Junit is a XML format, so if I ever go into here to refactor, my guess is that they Junit and XML formats may share a common base class or utility class in the future. So I figured I leave this here for now (essentially deferring the decision as to where things should go for now).